docs/source/cluster_management/sweep/targeted-sweep.rst

```diff
@@ -46,3 +46,31 @@ Targeted sweep uses a combination of :ref:`install and runtime AtlasDB configura
 
    ``enabled``, "false", "Whether targeted sweep should be run by background threads. Note that enableSweepQueueWrites must be set to true before targeted sweep can be run."
    ``shards``, "1", "Number of shards to use for persisting information to the sweep queue, enabling better parallelization of targeted sweep. The number of shards should be greater than or equal to the number of threads used for background targeted sweep. Note that this number must be monotonically increasing, and attempts to lower may be ignored. Maximum supported value is 256."
+
+Changing Sweep Strategy for a Table
+-----------------------------------
+
+.. warning::
+
+   Consult with the AtlasDB team before changing the sweep strategy of a table. Doing this incorrectly can invalidate
+   AtlasDB's correctness guarantees.
```

Comment:
We usually use `danger` and `SEVERE DATA CORRUPTION` for correctness violations, I think.